BenadrylAutomation/src/main/java/com/jnj/base/BaseClass.java

```diff
@@ -37,14 +35,11 @@ public class BaseClass {
 	public static ExtentTest test;
 	public String defaultFlag = "Yes";
 	public static String baseURI;
-	public static String runOn;
 	public ExtentReports exprep = ExtentManager.setExtent();
 
 	public static HeaderPage header;
 	public static HomePage home;
 	public static DifferencesPage difference;
```

Comment:
@zahinSJI i cannot see declaration of comapre and signup... why?

Reply:
They were added, they were not included for unknown reason